features/addons/pom.xml

```diff
@@ -1,77 +1,91 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-	<modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-	<parent>
-		<groupId>org.openhab.distro</groupId>
-		<artifactId>pom-features</artifactId>
-		<version>2.2.0-SNAPSHOT</version>
-	</parent>
+  <parent>
+    <groupId>org.openhab.distro</groupId>
+    <artifactId>pom-features</artifactId>
+    <version>2.2.0-SNAPSHOT</version>
+  </parent>
 
-	<artifactId>addons</artifactId>
-	<packaging>feature</packaging>
+  <artifactId>addons</artifactId>
+  <packaging>kar</packaging>
```

Comment:
We are in the "features" folder here, so you must not change packaging from feature to kar.

Reply:
See comment below, it has been done because you should not have to different feature repos (XML). The aggregator feature / KAR could be moved to dist easily.